java/src/main/native/OrtJniUtil.c

```diff
@@ -220,225 +221,207 @@ jfloat convertHalfToFloat(uint16_t half) {
 }
 
 jobject convertToValueInfo(JNIEnv *jniEnv, const OrtApi * api, OrtTypeInfo * info) {
```

Comment:
>OrtTypeInfo * info [](http://example.com/codeflow?start=63&length=18)

Should this be a const qualified so it is not modified and it is clear to be an input parameter?

Reply:
Sure.